### PR TITLE
TCI's data are set in the form

### PR DESCRIPTION
**Pull request by Ydrasil**
The data are correctly received in backend

### CODE REVIEW

**Comment by Ydrasil** [comment]
@vinyll

File: app.py [hunk 1]
```diff
@@ -11,7 +16,13 @@ def home():
 
 @app.route('/<string:token>')
 def survey(token):
-    return render_template('survey.html')
+    try:
+        verify_token = Token.objects.get(key=token, usage_date=None)
```
Comment (vinyll) [inline app.py]: `verify_token = ` is not used. Delete it.

File: templates/survey.html [hunk 2]
```diff
@@ -19,7 +19,62 @@
...
+  })
+
+  function end() {
+    document.querySelector('[name=times]').value = times;
+    document.querySelector('[name=answers]').value = answers;
+    const submitter = document.querySelector('#submitter')
+    return submitter.submit()
+  }
+
+  form.addEventListener('submit', event => {
```
Comment (vinyll) [inline templates/survey.html]: Move this up below the other `form.addEventListener`

**Review by vinyll — CHANGES_REQUESTED** [review]

File: templates/survey.html [hunk 3]
```diff
@@ -19,7 +19,62 @@
...
+  const form = container.querySelector('form')
+  const counterTag = container.querySelector('#counter')
+  const clickDelay = 500
+  let index = 0
+  let answers = []
+  let times = []
+
+  step(index)
+
+  function step(index) {
```
Comment (vinyll) [inline templates/survey.html]: move this down next to `function end()`

**Comment by Ydrasil** [comment]
@vinyll changes done

File: static/style.css [hunk 4]
```diff
@@ -113,6 +113,6 @@ button {
   text-align: right;
   font-style: italic;
 }
-[hidden] {
-  display: none !important;
+.incomplete button {
```
Comment (vinyll) [inline static/style.css]: Is this still useful? There's nomore button in the HTML

**Review by vinyll — CHANGES_REQUESTED** [review]

File: static/style.css (hunk 4 above)
Comment (Ydrasil) [reply static/style.css]: I've changed that

**Pull request closed without merging**